slices/libmagic1t64.yaml

```
@@ -0,0 +1,21 @@
+package: libmagic1t64
+
+essential:
+  - libmagic1t64_copyright
+
+slices:
+  libs:
+    essential:
+      - libbz2-1.0_libs
+      - libc6_libs
+      - liblzma5_libs
+      - libmagic-mgc_data
+      - zlib1g_libs
+    contents:
+      /etc/magic:
+      /etc/magic.mime:
+      /usr/lib/*-linux-*/libmagic.so.1*:
```

```suggestion
      /usr/lib/*-linux-*/libmagic.so.1*:
      /usr/share/misc/magic:
```

One more nitpick: This symlink is also missing.